### DISCUSSION TITLE
Difference between sandbox levels?

### DISCUSSION THREAD

**Discussion by WoxWik** [sandboxie-plus/Sandboxie#1744] [Documents and notes]
What exactly is the difference between the hardened sandbox levels which require payment and the default one?

Is the standard sandbox insufficient to run untrusted code?

**Comment by DavidXanatos** [comment]
So currently it looks like this
Hardened with data protection: exclusive
Hardened : available to anyone
Default with data protection: exclusive
Default : available to anyone
Compartment mode with data protection: exclusive
Compartment mode: also exclusive

The regular Hardened currently just enables drop admin rights and that's it,
and yes as things are dropping admin rights makes it considerably more difficult to exploit potential vulnerabilities.

That said with build 1.2.0 a new and improved Hardened mode will be introduced which will provide substantially improved isolation and be exclusive, like the other special modes.
